packages/docs/next-doc-site/components/code-preview-button.tsx

```diff
@@ -11,7 +11,7 @@ const buttonStyle = css`
   align-items: center;
   justify-content: space-between;
   padding: 6px 12px 6px 8px;
-  font-family: ${tokens.fontFamily.sansSerif};
+  font-family: ${tokens.fontFamily.sansSerif}, Arial, sans-serif;
```

Comment:
Nice

Reply:
Will the next version of the tokens include the fallbacks in the token?

Reply:
Yes